src/gateway/how-to/request-transformations.md

```diff
@@ -0,0 +1,115 @@
+but how could you provide a default value for your services in the meantime?
+
+[{{site.base_gateway}}](/gateway/{{page.kong_version}}/) supports a [Plugin](/hub/) 
+architecture including a [Request Transformer Plugin](/hub/kong-inc/request-transformer/) 
+that can modify incoming requests before proxying them to your upstream service. 
+This can all be accomplished using a no-code solution and managed with no downtime using 
+Kong's dynamic administrative capabilities.
+
+This guide will show you how to configure the Request Transformer plugin using 
+the [Kong Admin API](/gateway/{{page.kong_version}}/admin-api/) to modify incoming 
+requests with a static constant value. Then we will test the feature with a mock 
+request verifying the transformation process.
+
+### Prerequisites
+
+The guide assumes the following:
+
+* You have a {{site.base_gateway}} available for testing and is reachable on `localhost`. 
+If you would like help running a local gateway using Docker, see this companion 
+[{{site.base_gateway}} in minutes](/gateway/{{page.kong_version}}/how-to/kong-gateway/) guide. If your gateway is available on 
+a different host, adjust the commands in this guide appropriately.
+* You have a service named `mock` installed on your gateway. You can adjust the 
+commands to use a different service or use the [{{site.base_gateway}} in minutes](/gateway/{{page.kong_version}}/how-to/kong-gateway) guide to 
+run a gateway with a `mock` service.
+* You have [`curl`](https://curl.se/) installed on your system, which is used to send 
+requests to the gateway. Most systems come with `curl` pre-installed.
+* This guide uses the [`jq`](https://stedolan.github.io/jq/) command line JSON processing tool. While
+this tool is not necessary to complete the tasks, it's helpful for processing JSON responses from
+the gateway. If you do not have `jq` or do not wish to install it, you can modify the commands to remove
+`jq` processing.
+
+### Steps
+
+There are a large number of Kong plugins, many of which need to 
+be [custom installed](/gateway/{{page.kong_version}}/plugin-development/distribution/) 
+prior to utilization. Fortunately, Kong ships prepackaged with a number of useful plugins including
+the Request Transformer.
+
+First verify the Request Transformer plugin is available on your gateway by querying the Admin API and using `jq` to filter the response looking at the plugins available on the server.
+
+```sh
+curl -s http://localhost:8001 | \
+  jq -r '.plugins.available_on_server."request-transformer"'
+```
+
+The command output should be:
+```
+true
+```
+
+Now, assign a new instance of the Request Transformer plugin to
+the mock service by invoking the following `POST` request to the Admin API.
+In this command, the `config.add.body` value instructions the plugin to add a new
+field to the body of incoming requests before forwarding to the `mock` service.
+In this example, we are adding a field named `new-field`, and giving it a static value of
```

Comment:
Please see my revised version of this, I think the language is a little tricky, we are telling the plugin to do the work of adding a field, so i think it might be confusing to tell the user to add a field.  I tried to revise to clarify